sample/SampleHost/Program.cs

```diff
@@ -18,9 +19,17 @@ public static async Task Main(string[] args)
                 .ConfigureWebJobs(b =>
                 {
                     b.AddAzureStorageCoreServices()
-                    .AddAzureStorage()
-                    .AddServiceBus()
-                    .AddEventHubs();
+                    .AddAzureStorage();
+
+                    b.Services.AddOptions<ConcurrencyOptions>().Configure(options =>
```

Comment:
Won't be checked in, but shows how DC can be enabled/configured via options config